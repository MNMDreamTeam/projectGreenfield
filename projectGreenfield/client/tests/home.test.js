import React from 'react';
import {shallow} from 'enzyme';
import Home from '../src/components/home.jsx';
import VerticalCarousel from '../src/components/verticalCarousel';




describe('Home', () => {
  it('should render correctly with no props', () => {
    const component = shallow(<Home/>);
    
    expect(component).toMatchSnapshot();
  });
  // it('should render banner text correctly with given strings', () => {
  //   const id = 1;
  //   const component = shallow(<Home prodId={id} />);
  //   expect(component).toMatchSnapshot();
  // });
});
