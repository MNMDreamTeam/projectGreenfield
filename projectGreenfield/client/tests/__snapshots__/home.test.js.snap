// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render banner text correctly with given strings 1`] = `
<div>
  <div
    class="logoBar"
  >
    <img
      class="logoPic"
      src="./images/logo.jpg"
    />
  </div>
  <div
    class="Row"
  >
    <div
      class="showcase"
      id="Col"
    >
      <div />
      <div />
    </div>
  </div>
</div>
`;

exports[`Home should render carousel pictures correctly with given product id 1`] = `
<div>
  <div
    class="logoBar"
  >
    <img
      class="logoPic"
      src="./images/logo.jpg"
    />
  </div>
  <div
    class="Row"
  >
    <div
      class="showcase"
      id="Col"
    >
      <div />
      <div />
    </div>
  </div>
</div>
`;

exports[`Home should render correctly with no props 1`] = `
<div>
  <div
    class="logoBar"
  >
    <img
      class="logoPic"
      src="./images/logo.jpg"
    />
  </div>
  <div
    class="Row"
  >
    <div
      class="showcase"
      id="Col"
    >
      <div />
      <div />
    </div>
  </div>
</div>
`;
